### PR TITLE
New Buy Now workflow

### PR DESCRIPTION
**Pull request by jorgearellano19**
This PR contains the Buy Now workflow with the new design.

### CODE REVIEW

File: web/src/components/products/ProductActions/components/ProductPrice/ProductPrice.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+type Props = {
+  price: ProductPrice
+  unitsPerBox: number
+}
+const ProductPriceContainer: FC<Props> = ({ price, unitsPerBox }) => {
+  return (
+    <div className={s.container}>
+      <h1 className={s.price}>${price.amount}</h1>
+      <span className={s.text}>$ per unit</span>
```
Comment (eduardorr97) [inline web/src/components/products/ProductActions/components/ProductPrice/ProductPrice.tsx:14]: correct me if Im wrong, should we show 2 $ signs here?

File: web/src/components/products/ProductActions/components/CustomTestCard/index.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import CUstomTestCard from "./CustomTestCard"
+
+export default CUstomTestCard
```
Comment (eduardorr97) [inline web/src/components/products/ProductActions/components/CustomTestCard/index.ts:3]: Can we fix capitalization here

File: web/src/components/products/ProductActions/components/CustomTestCard/CustomTestCard.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import Links from "@components/common/Links"
+import s from "./CustomTestCard.module.scss"
+
+type Props = {
+  title: string
+  body: string
+  linkText: string
+}
+
+const CUstomTestCard: FC<Props> = ({ title, body, linkText }) => {
```
Comment (eduardorr97) [inline web/src/components/products/ProductActions/components/CustomTestCard/CustomTestCard.tsx:13]: Can we fix capitalization here?

File: web/src/components/products/ProductActions/components/ProductPrice/ProductPrice.tsx (hunk 1 above)
Comment (jorgearellano19) [reply web/src/components/products/ProductActions/components/ProductPrice/ProductPrice.tsx:14]: https://xd.adobe.com/view/b13b6bd4-f42e-47db-a8f7-70ef42752e9a-e468/screen/c5a0112a-8904-4f1e-aa65-9b31e679e38c/?x_product=cc-slack%2F1.5.0 That's the design

**Review by eduardorr97 — APPROVED** [review]
LGTM

**Review by diveddie — APPROVED** [review]
LGTM

**Pull request merged**